### PR TITLE
NodeOutOfServiceVolumeDetach feature update to GA

### PR DESCRIPTION
**Pull request by carlory**
Upstream: https://github.com/kubernetes/kubernetes/pull/118228
KEP: https://github.com/kubernetes/enhancements/pull/4029

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:295]:
```suggestion
| `NodeOutOfServiceVolumeDetach` | `true` | Beta | 1.26 | 1.27 |
```

Comment (Shubham82) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md]: please mention `1.27` for Beta  stage under `until` section
```suggestion
| `NodeOutOfServiceVolumeDetach` | `true` | Beta | 1.26 | 1.27 |
```

**Review by Shubham82 — CHANGES_REQUESTED** [review]

**Comment by Shubham82** [comment]
thanks, @carlory
/lgtm

**Comment by Shubham82** [comment]
It can be merged after the [PR:#118228](https://github.com/kubernetes/kubernetes/pull/118228) 
/hold

Please remove the hold label once the above PR will merge.

**Comment by YuikoTakada** [comment]
Thank you for your PR.
https://github.com/kubernetes/website/blob/main/content/en/docs/reference/command-line-tools-reference/feature-gates-removed.md seems need to be updated also.

**Comment by carlory** [comment]
This doc should only contains removed feature-gates. But the `NodeOutOfServiceVolumeDetach `still exists  in 1.28.

**Comment by YuikoTakada** [comment]
> This doc should only contain removed feature-gates. But the `NodeOutOfServiceVolumeDetach `still exists in 1.28.

Thank you for your reply. It was my misunderstanding, sorry. You don't need to update https://github.com/kubernetes/website/blob/main/content/en/docs/reference/command-line-tools-reference/feature-gates-removed.md . Thanks.

**Comment by YuikoTakada** [comment]
> Upstream: [kubernetes/kubernetes#118228](https://github.com/kubernetes/kubernetes/pull/118228) 
> KEP: [kubernetes/enhancements#3320](https://github.com/kubernetes/enhancements/pull/3320)

nit: correct KEP is https://github.com/kubernetes/enhancements/pull/4029/ ?

**Comment by carlory** [comment]
Thanks! It's updated.

File: content/en/docs/concepts/architecture/nodes.md
Comment (sftim) [inline content/en/docs/concepts/architecture/nodes.md:574]: (nit)
```suggestion
## Non-graceful node shutdown handling {#non-graceful-node-shutdown}
```

**Review by sftim — COMMENTED** [review]
/lgtm
for docs

**Comment by carlory** [comment]
@sftim @xing-yang please review again

**Pull request merged**
